torch/onnx/_internal/_exporter_legacy.py

```diff
@@ -433,9 +452,15 @@ def enable_fake_mode():
     )  # type: ignore[assignment]
 
 
+@deprecated(
+    "torch.onnx.dynamo_export is deprecated since 2.7.0. Please use torch.onnx.export(..., dynamo=True) instead.",
+)
 class ONNXRuntimeOptions:
     """Options to influence the execution of the ONNX model through ONNX Runtime.
 
+    .. deprecated:: 2.7
```

Comment:
Do we need to add it to the documentation since it is already flagged above?

Reply:
I think so. The flag is not rendered in the rendered docs.